### PR TITLE
Minor bugfixes

### PR DESCRIPTION
**Pull request by Oaphi**
The PR should fix the throttling issue when the bot sends multiple messages + connects `throttleSecs` to the multipart message handling

### CODE REVIEW

File: src/queue.js [hunk 1]
```diff
@@ -99,11 +99,11 @@ export const sendMultipartMessage = async (config, room, response, msg) => {
 
     if (numParts > maxMessageParts) {
         await msg.reply(`I wrote a poem of ${numParts} messages for you!`);
-        await wait(2);
```
Comment (samliew) [inline src/queue.js:102]: The reason why I set this to 1 & 2 seconds, is that we already pre-muted 2 seconds for each part (I just realised I should have used `numParts` instead), so we want to get the parts out as fast as possible within this mute.

Comment (Oaphi) [reply src/queue.js:102]: I just realized too I did not notice it was length of response and not the number of parts referenced in the `lastActivityTime` :(  Pushed.

Comment (Oaphi) [reply src/queue.js:102]: I don't see where we pre-muted, though, sorry, can you clarify? `lastActivityTime` does not affect the multipart message itself. The issue here is that we can only send 2 messages in short succession, then we land into chat's native throttle. Maybe the delay between part 1 and 2 could be 1 second and 2 for the rest?

**Review by samliew — APPROVED** [review]
For your review as well

File: src/config.js [hunk 2]
```diff
@@ -41,6 +50,11 @@ export class BotConfig {
     // Response when bot tries to post the exact same response again
     duplicateResponseText = "Please read my previous message...";
 
+    // Checks if the bot is currently muted
+    get isMuted() {
```
Comment (Oaphi) [inline src/config.js:54]: maybe make it a function to unify with how how we name `is*` methods? That's on me for later cleanup, looks good

**Pull request merged**